### ISSUE TITLE
Cannot lint custom file extension (e.g. test.scss.liquid)

### ISSUE THREAD

**Issue by gianpaj** [sasstools/sass-lint#1003]
**What version of Sass Lint are you using?**
```
$ sass-lint --version
1.10.2
```
**Please include any relevant parts of your configuration**
```
$ cat .sass-lint.yml
options:
  syntax: 'scss'
files:
  include: '**/*.scss.liquid'
```

**What did you do? Please include the actual source code causing the issue.**
```
$ sass-lint -c .sass-lint.yml assets/test.scss.liquid -v
Syntax "liquid" is not supported yet, sorry

assets/test.scss.liquid
  1:1  error  Cannot read property 'toString' of undefined  Fatal

✖ 1 problem (1 error, 0 warnings)


/usr/local/lib/node_modules/sass-lint/index.js:313
    throw new exceptions.SassLintFailureError(errorCount.count + ' errors were detected in \n- ' + errorCount.files.join('\n- '));
    ^
Error
    at Function.sassLint.failOnError (/usr/local/lib/node_modules/sass-lint/index.js:313:11)
    at detectPattern (/usr/local/lib/node_modules/sass-lint/bin/sass-lint.js:31:10)
    at /usr/local/lib/node_modules/sass-lint/bin/sass-lint.js:83:5
    at Array.forEach (native)
    at Object.<anonymous> (/usr/local/lib/node_modules/sass-lint/bin/sass-lint.js:82:16)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
```

```
$ cat assets/test.scss.liquid

/* -- Wishlist button collection -- */
$wk-button-collection-icon-color: #424242;
$wk-button-collection-backgrund-color: rgba(0,0,0,0.1);
$wk-button-collection-icon-size: 15px;
```

**What did you expect to happen?**
not to error out 😄 .
possible related ticket? #609 

**If you're using a IDE plugin have you tried the CLI too?**
Yes. Both are failing

**Comment by gianpaj** [comment]
By the way, that this works.
```
$ sass-lint -v -s scss assets/test.scss.liquid
```

But I'm looking for a way to configure sass-lint to work as I don't think I can pass a CLI parameter option to VS Code [extension](https://github.com/glen-84/vscode-sass-lint)
